VerificationErrors.dfy(8,38): Error: assertion might not hold
VerificationErrors.dfy(15,18): Error: assertion might not hold

Dafny program verifier finished with 0 verified, 2 errors
